openfl-workspace/tf_cnn_mnist/plan/data.yaml

```diff
@@ -0,0 +1,7 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# Licensed subject to the terms of the separately executed evaluation license agreement between Intel Corporation and you.
+
+# collaborator_name,data_directory_path
+one,1
```

Comment:
General: This has been conventionally done for quite some time in OpenFL, but we should transition to using more expressive names and paths to reduce users' cognitive overhead.

Reply:
agreed. actually, based on my understanding, the initial collaborator_name,data_directory_path in the data.yaml file is mainly for the aggregator to initialize with one of the data shards, since it is expected that the collaborators will update the path (and their name). With Patrick's new PR that allows users to specify input shape during plan initialization, this is actually not as necessary (I believe). Leaving it here for this PR though. Can do a sweep through and remove as needed later